app/enterprise/0.33-x/admin-gui/configuration/getting-started.md

```diff
@@ -12,7 +12,7 @@ Start Kong.
 
 `kong start -c kong.conf.default`
 
-> Note: Not all deployments of Kong utilize a configuration file, if this describes you (or you are unsure) please reference the [Kong configuration docs](https://getkong.org/latest/configuration/) in order to implement this step.
```

Comment:
This link works for me - if you visit https://docs.konghq.com/enterprise/0.33-x/admin-gui/configuration/getting-started/ and click that link, you are taken to https://docs.konghq.com/0.14.x/configuration/

However, I see two other issues now:

1. Kong Enterprise 0.33 is based on Kong 0.13.1, not on 0.14 - so, this `latest` URL points to the wrong version of Kong.

2. This whole sentence is unclear to me - what is it trying to say?